src/js/pie.js

```diff
@@ -245,4 +300,14 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     floe.chartAuthoring.pieChart.getViewBoxConfiguration = function (x, y, width, height) {
         return x + "," + y + "," + width + "," + height;
     };
+
+    // Returns a formatted string for a numeric data value based on a supplied template
+
+    floe.chartAuthoring.pieChart.getDataDisplayValueFromTemplate = function(that, template, d) {
```

Comment:
legend.js has the same function. You can create a common utils function for it to be shared.